mod/group_tools/pages/groups/invite.php

```diff
@@ -34,7 +35,7 @@
 		}
 		
 		elgg_push_breadcrumb(elgg_echo("groups"), "groups/all");
-		elgg_push_breadcrumb($group->name, $group->getURL());
+		elgg_push_breadcrumb(gc_explode_translation($group->name,true), $group->getURL());
```

Comment:
i don't think 2nd parameter for function gc_explode_translation is a boolean, it's either 'en' or 'fr' and spits out whatever language